PLATER/requirements.txt

```diff
@@ -7,7 +7,7 @@ starlette==0.14.2
 uvicorn==0.14.0
 reasoner-transpiler==1.9.0
 bmt-lite-2.1.0==2.1.1
-reasoner-pydantic==1.2.0.4
+reasoner-pydantic==1.2.0.1
```

Comment:
going backwards?

Reply:
constraints being defaulted to a list instead of None is causing problems downstream when transpling to cypher. We are looking into making transpiler make use of "constraints" attribute instead of other properties on the nodes and edges in the q_graph